SocialFilter/sections/specific.txt

```diff
@@ -62,6 +62,8 @@ refalover-note.mainichi.jp,note.calbee.jp,note.linemusic.jp,diehardtales.com,not
 !
+bugun.uz##.blog__share
+bugun.uz##.tg_link
```

Comment:
`bugun.uz##.tg_link` move this to `AdGuard Annoyances - annoyances.txt` under this section `Other annoyances - Social`.
You will see the rules with task comments.